core/spec/models/spree/shipment_spec.rb

```diff
@@ -24,6 +24,25 @@
   let(:variant) { mock_model(Spree::Variant) }
   let(:line_item) { mock_model(Spree::LineItem, variant: variant) }
 
+  context '#transfer_to_location' do
+    it 'transfers unit to a new shipment with given location' do
+      order = create(:completed_order_with_totals, line_items_count: 2)
+      shipment = order.shipments.first
+      variant = order.inventory_units.collect(&:variant).first
+
+      expect do
```

Comment:
It would be nice if we can wrap all these expectations in an `aggregate_failures` block so we can see multiple failures in a single run.

http://www.rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers%3Aaggregate_failures